sogs/hashing.py

```diff
@@ -0,0 +1,42 @@
+import nacl.hashlib
+from typing import Union, Iterable
+
+
+def blake2b(
+    data: Union[bytes, Iterable[bytes]],
```

Comment:
what if this was `Iterable` of things implementing the buffer interface or whatever the base type is called that is accepted by hasher.update?

Reply:
That will work fine as is, so just removed the type hint.